portfolio/src/main/webapp/index.html

```diff
@@ -8,28 +8,30 @@
   </head>
   <body>
     <div id="content">
-      <a href="index.html">Home</a>
-      <p><a href="about-me.html">About Me</a> -- Click here to learn more about me</p>
-      <p><a href="baking.html">Macarons and Things</a> -- A food gallery of my best baking accomplishments</p>
+      <ul>
+        <li><a href="index.html" class="active">Home</a></li>
+        <li><a href="about-me.html">About Me</a></li>
+        <li><a href="baking.html">Food</a></li>
+      </ul>
       <h1>Megan Shi</h1>
-      <img src="images/me.JPG" height=250 alt="Photo of Me">
+      <img src="images/me.JPG" height=250 alt="Photo of Me" class="center">
       <p>Hi, I'm Megan! I'm a rising junior at the University of Washington studying CS. 
         I like food (making AND eating) and love trying new recipes. In my free time, 
         I like playing tennis with my mom and video games with my friends. I'm looking forward to learning
         about software engineering and building projects. I also hope to build a PC this 
         summer so if you have any guidance let me know :^)</p>
       <p>
       <p>I also have two cats named Buzz and Ozzy!</p>
-      <img src="/images/cats.JPG" id="cat-image" height=250 alt="My cats Buzz and Ozzy">
-      <div>
-        <button onclick="changeCatPhoto()">click me to get a random photo of my cats!</button>
+      <img src="/images/cats.JPG" id="cat-image" height=250 alt="My cats Buzz and Ozzy" class="center">
+      <div class="centered-button">
+        <button class="button" onclick="changeCatPhoto()">click me to get a random photo of my cats!</button>
```

Comment:
Would you want these styles on the button class applied to all buttons? 

If so could you change the CSS to `button {...}` (no `.`) instead? That way you could do `<button>` instead of `<button class=...>`

And then, if you want to override styling or add additional styling to a special button, you could define a class styling for that one specifically.

Depending on what you're going for, the same concept could apply to images. If you'd like all images to be centered, I'd apply the styling to the element. Then for special cases (like the icon) you could define a class to apply special styling.

What do you think?

Reply:
This makes sense. I didn't really consider it because I only have one button in my website, but if I were to add more I would definitely want them to generally have the same style. Thanks!